pkg/azuredisk/azuredisk_v2.go

```diff
@@ -89,6 +92,31 @@ const (
 	azDriverNodeHealthy      azDriverNodeStatus = "Driver node healthy."
 )
 
+// Metrics subsystem and all of the keys used by the rest client.
+const (
+	RestClientSubsystem = "rest_client"
+	LatencyKey          = "throttling_request_latency_seconds" // edits: prefix or postfix
```

Comment:
Yes, I think we should.

Reply:
Also, since we are using an abstraction over the rest client in our code, I am thinking that maybe we should rename `rest_client` to `kube_client`? And maybe `throttling_request_latency_seconds` to `rate_limiter_hrottling_request_latency_seconds`?